fix resolve pipelinespec

chmouel opened the pull request:
- Fix linting error  in payload Replace
- Resolve pipelinespec embedded in PipelineRun too
- Fix others \r\n from triggers/github

chmouel committed:
Fix linting error  in payload Replace

pkg/webvcs/github.go:46:12: wrapperFunc: use strings.ReplaceAll method
in `strings.Replace(payload, "\r\n", "", -1)` (gocritic)
	payload = strings.Replace(payload, "\r\n", "", -1)

chmouel committed:
Resolve pipelinespec embedded in PipelineRun too

chmouel committed:
Fix others newlines errors when parsing the json

Signed-off-by: Chmouel Boudjnah <chmouel@redhat.com>

vdemeester commented on pkg/resolve/resolve.go:
```diff
@@ -29,7 +29,8 @@ func readTypes(data []byte) (Types, error) {
 		if err != nil {
-			return types, err
+			// If it's not a Kubernetes style yaml just ignore it.
+			continue
```
Do we want to log the error somehow ?

chmouel replied:
Yep maybe we should..

chmouel committed:
Don't create status if we have bad token

Or it will crash.

chmouel committed:
Log when there is a non tekton/kubernetes templates in .tekton

If we have a non tekton/kubernetes template in yaml we were just skipping, now
we are log.info it.

chmouel commented on pkg/cmd/pipelineascode/pipelineascode.go:
```diff
@@ -60,7 +60,7 @@ func runWrap(p cli.Params, opts *pacpkg.Options) error {
 
 	err = pacpkg.Run(cs, runInfo)
-	if err != nil {
+	if err != nil && !strings.Contains(err.Error(), "403 Resource not accessible by integration") {
```
That should only happen on dev test and messing around with INSTALLATION_ID number
